modules/createBowerPackage.js

```
@@ -0,0 +1,55 @@
+import { join as joinPaths} from 'path'
+import { stat as statFile, readFile, createWriteStream } from 'fs'
+import getProperty from './getProperty'
+import archiver from 'archiver'
+
+
+const promisify = func => (...args) =>
```

> If you're not extremely careful, it's easy to swallow errors. After years of using them, I still occasionally accidentally wrote code that swallowed errors.

I agree defaulting promises to swallow errors was a horrible design decision, but it *can* be solved with tooling. Chrome DevTools reports unhandled rejections, and for Node.js you could use [`loud-rejection`](https://github.com/sindresorhus/loud-rejection).